frontend/src/views/Products.ListProducts/Products.ListProducts.tsx

```diff
@@ -108,38 +108,11 @@ const ListProducts: React.VC = ({ verboseName, parentName }) => {
     fetchProducts();
   };
 
-  const onOrderFailed = () : void => {
-    notification.error({
-      message: '¡Ocurrió un error al intentar generar la orden de compra!',
-      description: 'Intentalo después.',
-    });
-  };
-
   const createOrder = async () : Promise<void> => {
     setLoading(true);
     if (productsSh.length > 0) {
-      const [, error] = await backend.orders.createOne({
-        productsSh,
-      });
-      if (error) {
-        onOrderFailed();
-        history.replace('/productos/');
-      } else {
-        notification.success({
-          message: '¡Petición de orden creado exitosamente!',
-          description: 'Su orden de compra ha sido recibida y será procesada.'
-            + 'El proveedor le informará lo mas pronto posible '
-            + 'el estatus de su solicitud.',
-        });
-        history.replace('/pedidos/cliente');
-      }
-    } else {
-      notification.error({
-        message: '¡Carrito de compras vacío!',
-        description: 'Inserte alguna cantidad en algún producto que requiera',
-      });
+      history.replace('/pedidos');
```

Comment:
A lo mejor cambiar a history.push para que el usuario pueda retroceder a la lista de productos.

```suggestion
      history.push('/pedidos');
```